processor/cumulativetodeltaprocessor/config.go

```diff
@@ -24,15 +24,19 @@ import (
 type Config struct {
 	config.ProcessorSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// List of cumulative sum metrics to convert to delta
+	// List of cumulative metrics to convert to delta. Default: converts all cumulative metrics to delta.
 	Metrics []string `mapstructure:"metrics"`
+
+	// The total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely.
+	MaxStale time.Duration `mapstructure:"max_stale"`
+
+	// Set to false in order to convert non monotonic metrics
+	MonotonicOnly bool `mapstructure:"monotonic_only"`
```

Comment:
What would be the default behavior here?

Reply:
The default behavior is to convert only monotonic metrics. (defaults to true)
My understanding is that non-monotonic sum metrics are treated as gauges and therefore should not be converted by default 😄